docs/source/terms_and_model.rst

```diff
@@ -353,17 +353,13 @@ A contiguous region specified by named features.
 
 **Implementation guidance**
 
-* `start` and `end` attributes of NamedInterval are intentionally
+* `start` and `end` attributes of CytobandInterval are intentionally
   specified vaguely in order to accommodate a wide variety of
   uses. Examples include named markers on chromosomes, cytogenetic
   bands, and legacy marker names found in older scientific literature.
```

Comment:
What are legacy marker names? If we are constraining the name of the class to cytoband, we might want to consider updating this list of examples.

Reply:
My intent was to capture old named markers (usually a format like `D13S45`, but probably lots more). These are merely names of probes that were used for ISH/FISH. However, I just discovered that NCBI dropped the probe database. So, my "legacy" might actually have become "long-forgotten and irrelevant". I'll drop that phrase because it doesn't add anything.